config/proxmox.php

```diff
@@ -0,0 +1,19 @@
+<?php
+Configure::set('Proxmox.email_templates', [
+    'en_us' => [
+        'lang' => 'en_us',
+        'text' => 'Thank you for ordering VPS 1, details below:
```

Comment:
VPS 1?

Reply:
I think it's a placeholder for the package name, I copied this from the user manual. But I think "Thank you for ordering your VPS, details below" would be better.